content/docs/100-getting-started/index.md

```diff
@@ -0,0 +1,3 @@
+---
```

Comment:
The naming with numbering system is not maintainable. However, we've been talking internally about what to do with this site, so again, wouldn't change anything now. I'm fine working within the system just to get us up and running.